routes/index.js

```diff
@@ -45,12 +46,22 @@ exports = module.exports = (app) => {
   viewRouter.use((req, res, next) => {
     // Extract locale (e.g.) 'en' or 'chn' from full url
     const locale = req.originalUrl.split('/')[1];
+
+    // If /en/ or /chn/ not included in url
     if ([ 'en', 'chn', ].includes(locale) != true) {
```

Comment:
Just a suggestion: it be better if we don't always hardcode the strings representing the URLs. What if we make a file in the `/setup` folder called `locales.js` and keep this array in there?